Cargo.toml

```diff
@@ -5,6 +5,8 @@ members = [
     "github/slack/issue-notifier/rust",
     "github/slack/commit-syncer/rust",
     "github/slack/pr-messager/rust",
+    "github/slack/label-changed/rust",
+    "github/slack/branch-changed/rust",
```

Comment:
Can you remove this invalid member path?

Reply:
sure, will do